saleor/attribute/models/base.py

```diff
@@ -215,6 +215,86 @@ def get_translated_keys(self):
         return {"name": self.name}
 
 
+class AttributeValueManager(models.Manager):
```

Comment:
This class could use some comments. I remember debugging another manager somewhere else and it was quite hard to follow along. E.g. here it's not clear upon first look what's happening with the nested `for` in `bulk_get_or_create` 🙌